installing/installing_vsphere/preparing-to-install-on-vsphere.adoc

```diff
@@ -0,0 +1,76 @@
+[id="preparing-to-install-on-vsphere"]
+= Preparing to install on vSphere
+include::modules/common-attributes.adoc[]
+:context: preparing-to-install-on-vsphere
+
+toc::[]
+
+
+[id="preparing-to-install-on-vsphere-prerequisites"]
+== Prerequisites
+
+* You reviewed details about the xref:../../architecture/architecture-installation.adoc#architecture-installation[{product-title} installation and update] processes.
+
+* You read the documentation on xref:../../installing/installing-preparing.adoc#installing-preparing[selecting a cluster installation method and preparing it for users].
+
+* You provisioned xref:../../storage/understanding-persistent-storage.adoc#understanding-persistent-storage[persistent storage] for your cluster. To deploy a private image registry, your storage must provide
+`ReadWriteMany` access modes.
+
+* If you use a firewall and plan to use Telemetry, you
+xref:../../installing/install_config/configuring-firewall.adoc#configuring-firewall[configured the firewall to allow the sites] required by your cluster.
+
+* For user-provisioned infrastructure, xref:../../installing/installing-mirroring-installation-images.adoc#installing-mirroring-installation-images[you created a registry on your mirror host] and obtained the `imageContentSources` data for your version of {product-title}.
+
+[IMPORTANT]
+====
+The steps for performing a user-provisioned infrastructure installation are provided as an example only. Installing a cluster with infrastructure you provide requires knowledge of the vSphere platform and the installation process of {product-title}. Use the user-provisioned infrastructure installation instructions as a guide; you are free to create the required resources through other methods.
+====
+
+[NOTE]
+====
+Be sure to also review this site list if you are configuring a proxy.
+====
+
+include::modules/cluster-entitlements.adoc[leveloffset=+1]
+
+include::modules/installation-vsphere-infrastructure.adoc[leveloffset=+1]
```

Comment:
If there is a need to duplicate this in this "Preparing to install" page (i.e. in addition to it being in the installation assemblies), then my view is that this should be consistent across the equivalent page in the other platform subsections.

Reply:
Removed. Thank you!